packages/http-client-java/generator/http-client-generator-standard-test/src/test/java/payload/pageable/PageableTests.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package payload.pageable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import payload.pageable.serverdrivenpagination.LinkResponse;
+import payload.pageable.serverdrivenpagination.LinkResponseLinks;
+
+public class PageableTests {
+  private final PageableClient client = new PageableClientBuilder().buildPageableClient();
+
+  @Test
+  public void testLink() {
```

Comment:
This test will not pass, as we don't have support in clientcore yet. (the next page cannot be retrieved)

But fine to keep.

Reply:
The version has been rollback to commit#f385033